internal/components/azaks/executor.go

```diff
@@ -91,3 +99,35 @@ func (e *AksOperationsExecutor) ExecuteSpecificCommand(operation string, params
 
 	return e.Execute(newParams, cfg)
 }
+
+// validateArguments performs early validation of arguments to prevent command injection
+func (e *AksOperationsExecutor) validateArguments(args string) error {
+	// Define dangerous patterns that could be used for command injection
+	dangerousPatterns := []string{
+		";",  // Command separator
+		"|",  // Pipe
```

Comment:
suggest doing some test on control plane logs - it has kql that has '|‘ in it.

Reply:
Oh, you're right. There is already command validations. Let me close this PR.